pages/common/jar.md

```diff
@@ -0,0 +1,7 @@
+# jar
+
+> Tool empowering work with *.jar files, used to distribute application software or libraries on the Java platform.
+
+- Unzip *.jar/*.war file to the current directory:
```

Comment:
I know this example was different in a previous commit of yours. Could you change this to reflect the changes to the command? Like "Extract all wars to the...", you know the lingo.